src/Cell.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class Cell extends React.Component{
+  constructor(props){
+    super(props);
+    this.startGame = this.startGame.bind(this);
+    this.getKeyDirection = this.getKeyDirection.bind(this);
+    this.foodPosition = 50;
+    this.state = { color : 'white', snakeHead: 0, direction: ''};
+  }
+  getKeyDirection(event){
```

Comment:
It is not a hard coding, the events KeyCodes is to get the direction of the arrowKeys to control the snake when pressed.